vue-frontend/src/components/contact/NewContactForm.vue

```diff
@@ -0,0 +1,576 @@
+            >
+              <span
+                class="tw-mr-6 tw-text-black-core/[0.6] tw-font-inter-regular tw-text-[1rem] md:tw-text-[1.375rem] tw-leading-[2.21rem] md:tw-leading-[3.0625rem]"
+                @click="toggleNDA"
+                >Send me NDA</span
+              >
+
+              <label
+                for="nda"
+                class="tw-relative tw-inline-flex tw-items-center tw-cursor-pointer"
+              >
+                <input
+                  name="nda"
+                  id="nda"
+                  type="checkbox"
+                  v-model="NDA"
+                  class="tw-sr-only tw-peer"
+                />
+                <div
+                  class="tw-w-9 tw-h-4 tw-rounded-full tw-bg-slate-400 tw-peer after:tw-content-[''] after:tw-absolute after:tw-top-[-2px] after:tw-left-[-2px] after:tw-h-5 after:tw-w-5 after:tw-border-slate-400 after:tw-border after:tw-rounded-full after:tw-bg-white after:tw-transition-all peer-focus:tw-outline-none peer-checked:after:tw-border-slate-400 peer-checked:tw-bg-gradient-to-r peer-checked:tw-from-[#F2709C] peer-checked:tw-to-[#FF9472] peer-checked:after:tw-translate-x-full"
+                ></div>
+              </label>
+            </div>
+          </div>
+
+          <div class="tw-pt-6">
+            <div
+              class="tw-text-black-core/[0.6] tw-font-inter-regular tw-text-[1rem] md:tw-text-[1.375rem] tw-leading-[1.21rem] md:tw-leading-[2.0625rem]"
+            >
+              What's your preferred mode of communication?
+            </div>
+            <div
+              class="tw-flex tw-flex-row tw-space-x-2 tw--ml-4 md:tw-ml-0 tw-w-fit tw-pt-4"
+            >
+              <label class="contactLabel1"
+                ><input
+                  class="tw-hidden"
+                  type="radio"
+                  name="contact_type"
+                  value="2"
+                  v-model.number="contactType"
+                  :disabled="disableInput"
+                />
+                <div
+                  class="tw-flex tw-items-center tw-mb-6 sm:tw-mr-9 sm:tw-mb-0 tw-cursor-pointer tw-rounded-full tw-px-6 tw-py-2 md:tw-py-2.5 active:tw-scale-[0.98] tw-font-inter-semibold tw-text-[1rem] md:tw-text-[1.1875rem] tw-leading-[1.21rem] md:tw-leading-[1.386875rem] tw-rounded-full tw-p-3 tw-text-center gradient-border-btn gradient-btn"
+                >
+                  <font-awesome-icon
+                    class="fa md:tw-mr-2 tw-w-[22.99px] md:tw-w-[28.18px] tw-h-[15.83px] md:tw-h-[24.18px] tw-text-[#f2709c]"
+                    :icon="phone"
+                    aria-hidden="true"
+                  />
+                  <span class="v2-canopas-gradient-text">Call</span>
+                </div></label
+              >
+
+              <label class="contactLabel2"
```

Comment:
Space between call and mail button should be less according to figma

Reply:
need to look on spacing between this buttons in all devices and they should be in center in mobile devices